src/Dropdown/Dropdown.jsx

```diff
@@ -240,13 +242,13 @@ const Toggler = styled.button(
     padding: 3,
     display: 'block',
     width: '100%',
-    background: 'transparent',
-    border: 'none',
+    background: colors.backgroundLightGray,
     outline: 'none',
     borderWidth: 1,
     borderStyle: 'solid',
-    borderColor: 'outline',
-    borderRadius: 8,
+    borderColor: colors.inputGray,
```

Comment:
Please have a meaningful name for colours:
```suggestion
    borderColor: colors.inputBorderColor,
```